src/components/ScreenWrapper.js

```diff
@@ -46,6 +46,7 @@ const ScreenWrapper = props => (
             return (
                 <View
                     style={[
+                        ...props.style,
                         styles.flex1,
                         paddingStyle,
```

Comment:
This may create duplicate styles. We can use [_.union](https://underscorejs.org/#union) to avoid this. We've already included it in our package.json, so you can just `import _ from 'underscore';` and use it here.

Reply:
Oh yeah.